WorldCleanup/Avatar.cs

```diff
@@ -49,12 +50,16 @@ internal static class Parameters {
             "VRCFaceBlendV",
         };
 
-        public static List<AvatarParameter> FilterDefaultParameters(Il2CppSystem.Collections.Generic.Dictionary<int, AvatarParameter>.ValueCollection src) {
+        public static List<AvatarParameter> FilterDefaultParameters(List<AvatarParameter> AvatarParameterList) {
             /* Note: IL2CPP Dictionary misses "Which" */
```

Comment:
System.List<AvatarParameter> would have a Which extension so you could have use that here.